src/index.js

```diff
@@ -72,16 +74,16 @@ async function handleEvent(event) {
  * route on a zone, or if you only want your static content
  * to exist at a specific path.
  */
-function handlePrefix(prefix) {
-  return (request) => {
-    // compute the default (e.g. / -> index.html)
-    let defaultAssetKey = mapRequestToAsset(request);
-    let url = new URL(defaultAssetKey.url);
+// function handlePrefix(prefix) {
```

Comment:
Why is all this commented out?

Reply:
It was a function that wasn't being used so eslint flagged it as an error.